bcda/cclf/cclf.go

```diff
@@ -67,22 +67,27 @@ func importCCLF0(fileMetadata *cclfFileMetadata) (map[string]cclfFileValidator,
 	)
 
 	var validator map[string]cclfFileValidator
-	for i, f := range r.File {
-		fmt.Printf("Reading file #%d from archive %s.\n", i, fileMetadata)
-		log.Infof("Reading file #%d from archive %s", i, fileMetadata)
-		if err := validateFileName(f.Name); err != nil {
-			fmt.Printf("Unknown file name when validating file: %s.\n", f.Name)
-			err = errors.Wrapf(err, "unknown file name when validating file: %s.\n", f.Name)
-			log.Error(err)
-			return nil, err
+	var rawFile *zip.File
+
+	for _, f := range r.File {
+		// needed for split files, otherwise this could have been an '==' check
```

Comment:
What are "split files"?  Why can't this be an `==` check?

Reply:
Split file logic was there before, i personally didn't understand why we needed it (like if we can't trust them to deliver files...), but i only focused on changing the file format acceptance logic and nothing else.